buildengine/build_java.go

```diff
@@ -20,7 +20,9 @@ func buildJavaModule(ctx context.Context, module Module) error {
 		logger.Warnf("unable to update ftl.version in %s: %s", module.Config.Dir, err.Error())
 	}
 	logger.Infof("Using build command '%s'", module.Config.Build)
-	err := exec.Command(ctx, log.Debug, module.Config.Dir, "bash", "-c", module.Config.Build).RunBuffered(ctx)
+	command := exec.Command(ctx, log.Debug, module.Config.Dir, "bash", "-c", module.Config.Build)
+	command.Env = append(command.Env, "FTL_MODULE_NAME="+module.Config.Module)
```

Comment:
Won't this mean you won't be able to build with plain old Maven without setting this envar? Can we not use a pom.xml property instead?

Reply:
It will default to the artifactId from the pom.xml for now if it is missing (which is what it is doing now). I do need a better solution for this, but this change means that at least in some circumstances it will actually use the value from the file.

Reply:
I think this is a minor improvement on the current state of affairs, there is already an issue about this that I will leave open to fix later.